.github/workflows/gh-pages.yml

```diff
@@ -10,7 +10,7 @@ on:
     # Match only on specific tags. We don't want this workflow to be invoked when
     # we put sliding `v{major}` and `v{major}.{minor}` tags on the same commit.
     tags: ['v[0-9]+.[0-9]+.[0-9]+*']
-    workflow_dispatch:
+  workflow_dispatch:
```

Comment:
This seems like an accidental change

Reply:
Nope, I should have left a comment on the PR. The `workflow_dispatch` item is currently under the `push` action, which means that it can't be triggered manually. This moves it under the `on:` item

Reply:
Ah, I see. I suppose github would show a warning if this workflow is run

Reply:
Imagine having a tool to statically analyze source code or files. That would be a cool invention :wink: xD